### PR TITLE
Add provider binary installation scripts

### PR DESCRIPTION
**Pull request by joannelynch92**
Update INSTALL.md with link to each script

### CODE REVIEW

**Comment by joannelynch92** [comment]
You can see the diff from the old scripts here https://github.com/HPE/terraform-provider-hpegl/pull/239/files (that PR complained about conflicts in the files)

File: scripts/install-hpegl-provider-macos.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  echo 'Intel Architecture detected'
+  arch="amd64"
+elif [[ `uname -m` == 'arm64' ]] 
+then
+  echo 'Apple Silicon Architecture detected'
+  arch="arm64"
+fi
+
+repo="HPE/terraform-provider-hpegl"
+macos_hpegl_dir="${HOME}/.terraform.d/plugins/local/hpe/hpegl"
```
Comment (eamonnotoole) [inline scripts/install-hpegl-provider-macos.sh:18]: That's not going to work I'm almost certain.  You need `registry.hashicorp.io/hpe/hpegl`

File: scripts/install-hpegl-provider-windows.ps1 [hunk 2]
```diff
@@ -0,0 +1,53 @@
+param ($VERSION)
+
+$os="windows"
+$arch="amd64"
+$repo="HPE/terraform-provider-hpegl"
+$windows_hpegl_dir="$env:appdata\terraform.d\plugins\local\hpe\hpegl"
```
Comment (eamonnotoole) [inline scripts/install-hpegl-provider-windows.ps1:6]: Same here.

File: scripts/install-hpegl-provider.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+os="linux"
+arch="amd64"
+repo="HPE/terraform-provider-hpegl"
+linux_hpegl_dir="${HOME}/.local/share/terraform/plugins/local/hpe/hpegl"
```
Comment (eamonnotoole) [inline scripts/install-hpegl-provider.sh:8]: Same here.

**Review by eamonnotoole — COMMENTED** [review]
Joanne, I think that the local cache paths are wrong.  Could you double check and test by removing all existing local cache first?

File: scripts/install-hpegl-provider-macos.sh (hunk 1 above)
Comment (joannelynch92) [reply scripts/install-hpegl-provider-macos.sh]: Ok will revert to what the original script uses - registry.terraform.io. This works for me.

**Review by eamonnotoole — APPROVED** [review]
Very nice.

**Pull request merged**